### ISSUE TITLE
Discrepancy among mAP scores on DarkFace validation dataset

### ISSUE THREAD

**Issue by aasharma90** [cuiziteng/ICCV_MAET#15]
Hi, 
Thanks for releasing your code and pre-trained models. 

I'm observing some inconsistency in the results on the DarkFace validation set. Here are the details -
1) On running `test.py` with the pre-trained model, I can get an mAP score of `0.5619` 
2) In the paper, the mAP score reported is `0.5580`
3) Running the official [DarkFace Evaluation tool](https://github.com/Ir1d/DARKFACE_eval_tools), I get an mAP score of `0.4421`

For 3) above, here is a snapshot of the run -
![maet_run](https://user-images.githubusercontent.com/27413823/221070677-835686cc-03c0-4a43-b375-d37e2f6ad4b0.png)

For 3), I have also followed the recommended [evaluation guidelines](http://www.ug2challenge.org/rules19_t2.html). For your reference, please find sample `gt` and `outputs` files below respectively -
[9.txt](https://github.com/cuiziteng/ICCV_MAET/files/10819916/9.txt)
[9.txt](https://github.com/cuiziteng/ICCV_MAET/files/10819918/9.txt)

Would greatly appreciate if you could please help me resolve the discrepancy in the results (and let me know which one should be the official result?)

Regards!

**Comment by cuiziteng** [comment]
Hello, thanks to the questions.

For the official [DarkFace Evaluation tool](https://github.com/Ir1d/DARKFACE_eval_tools), I think this may caused by the different version or different dataset setting divide? 

In addition you could use the result for comparison if it suitable in the setting, that's totally OK. 

Regards ~

**Comment by aasharma90** [comment]
If you'd like any further detail, please let me know. Thanks!

**Comment by aasharma90** [comment]
Hi, thanks for your quick response!

Would it be possible for you to kindly run the [DarkFace Evaluation tool](https://github.com/Ir1d/DARKFACE_eval_tools) on the DarkFace validation dataset with your pre-trained model at your end?

Just want to make sure that there is no wrong settings at my end. Would greatly appreciate your help!

**Comment by cuiziteng** [comment]
Hi ~ I would try it later and give you the response, cause I'm rushing for the paper deadline now.

Sorry for that and thanks for the understanding.

**Comment by aasharma90** [comment]
Sure, no worries. Thank you!
